src/libcore/num/f32.rs

```diff
@@ -72,9 +72,17 @@ pub mod consts {
     // FIXME: replace with mathematical constants from cmath.
 
     /// Archimedes' constant (π)
+    ///
+    /// Equal to τ/2.
```

Comment:
Regardless of whether we should add tau to the standard library or not, I don't think that this is a worthwhile addition to the doc comment of pi. Pi is well-known among the general population, you won't have to look far to find mathematicians who don't know about tau.

Reply:
Sure. Removed it.